packages/react-core/src/components/Chip/examples/Chip.md

```diff
@@ -2,13 +2,32 @@
 id: Chip
 section: components
 cssPrefix: pf-c-chip
-propComponents: ['Chip']
+propComponents: ['Chip', 'ChipGroup']
 ouia: true
 ---
 
-## Examples
+## Chip examples
 
-### Default
+### Variants
+Chips can be removable or read-only. The Overflow chip is a special chip that is used to expand or collapse the content of a chip group.
 
 ```ts file='./ChipDefault.tsx'
 ```
+
+## Chip group examples
+A chip group is a collection of chips that can be grouped by category and used to represent one or more values assigned to a single attribute. When the value of `numChips` is exceeded, additional chips will be hidden using an overflow chip.
```

Comment:
```suggestion
```

Move this to the simple inline chip groups example to better indicate that all of the examples are part of the "chips" component?